src/services/news/events/index.js

```diff
@@ -1,8 +1,8 @@
 const logger = require('winston');
 
-const removeTeamNews = (app, newsService) => async (deletedTeam) => {
+const removeTeamNews = app => async (deletedTeam) => {
 	try {
-		await newsService.remove(null, {
+		await app.service('/newsModel').remove(null, {
```

Comment:
note für mich

Reply:
Da es sich hier um eine bulk-Operation handelt und ich keinen User in der Hand habe, muss ich hier den ModelService benutzen, aber nicht das Model direkt.